sbi/user_input/user_input_checks.py

```diff
@@ -464,10 +457,10 @@ def prepare_for_sbi(simulator: Callable, prior,) -> Tuple[Callable, Distribution
 
     One of the goals is to allow you to use sbi with inputs computed in numpy.
 
-    Attempts to meet the following requirements by reshaping and type-casting:
-    - the simulator function receives as input and returns a Tensor.
-    - the simulator can simulate batches of parameters and return batches of data.
-    - the prior does not produce batches and samples and evaluates to Tensor.
+    Attempts to meet the following requirements by reshaping and type-casting:<br/>
```

Comment:
Perhaps the list is parsed properly without hard line breaks `<br/>` by simply leaving an empty line after the colon?

Reply:
Yes, that works! And it even renders the list in a prettier way :)